csharp/Clean Code/ToDo/Program.cs

```diff
@@ -5,106 +5,88 @@ namespace ToDo
         }
-        /// <summary>
-        /// Show the main menu 
-        /// </summary>
-        /// <returns>Returns option indicated by user</returns>
-        public static int ShowMainMenu()
+
+        public static int ShowMainMenuAndGetChoice()
         {
             Console.WriteLine("----------------------------------------");
             Console.WriteLine("Enter the option to perform: ");
             Console.WriteLine("1. New task");
             Console.WriteLine("2. Remove task");
             Console.WriteLine("3. Pending tasks");
             Console.WriteLine("4. Exit");
-
-            // Read line
-            string line = Console.ReadLine();
-            return Convert.ToInt32(line);
+            
+            return Convert.ToInt32(Console.ReadLine());
         }
 
-        public static void ShowMenuDos()
+        public static void AddTask()
         {
-            try
-            {
-                Console.WriteLine("Enter the number of the task to remove: ");
-                // Show current taks
-                for (int i = 0; i < TL.Count; i++)
-                {
-                    Console.WriteLine((i + 1) + ". " + TL[i]);
-                }
-                Console.WriteLine("----------------------------------------");
+            Console.WriteLine("Enter the name of the task: ");
+            string task = Console.ReadLine();
 
-                string line = Console.ReadLine();
-                // Remove one position
-                int indexToRemove = Convert.ToInt32(line) - 1;
-                if (indexToRemove > -1)
-                {
-                    if (TL.Count > 0)
-                    {
-                        string task = TL[indexToRemove];
-                        TL.RemoveAt(indexToRemove);
-                        Console.WriteLine("Task " + task + " deleted");
-                    }
-                }
-            }
-            catch (Exception)
-            {
-            }
+            Tasks.Add(task);
+
+            Console.WriteLine("Task registered successfully");
         }
 
-        public static void ShowMenuAdd()
+        public static void RemoveTask()
         {
-            try
-            {
-                Console.WriteLine("Enter the name of the task: ");
-                string task = Console.ReadLine();
-                TL.Add(task);
-                Console.WriteLine("Task registered successfully");
-            }
-            catch (Exception)
+            Console.WriteLine("Enter the number of the task to remove: ");
+
+            ShowCurrentTasks();
+
+            int indexToRemove = Convert.ToInt32(Console.ReadLine()) - 1;
+
+            // Validate and remove one position
+            if (indexToRemove > -1)
             {
+                if (Tasks.Count > 0)
+                {
+                    string task = Tasks[indexToRemove];
+                    Tasks.RemoveAt(indexToRemove);
+                    Console.WriteLine("Task " + task + " deleted");
```

Comment:
You should use string interpolation for a better coding style
```suggestion
                    Console.WriteLine($"Task {task} deleted");
```